Makefile

```diff
@@ -152,6 +160,40 @@ proto-gen: proto-build ## Generate all the code from the Protobuf files
 	@cp -r github.com/okp4/okp4d/x/* x/
 	@rm -rf github.com
 
+## Release:
+release-assets: release-binary-all release-checksums ## Generate release assets
+
+release-binary-all: $(RELEASE_TARGETS)
+
+$(RELEASE_TARGETS): ensure-buildx-builder
+	@GOOS=$(word 3, $(subst -, ,$@)); \
+    GOARCH=$(word 4, $(subst -, ,$@)); \
+    BINARY_NAME="okp4d-${VERSION}-$$GOOS-$$GOARCH"; \
+	echo "${COLOR_CYAN} 🎁 Building ${COLOR_GREEN}$$GOOS $$GOARCH ${COLOR_CYAN}release binary${COLOR_RESET} into ${COLOR_YELLOW}${RELEASE_FOLDER}${COLOR_RESET}"; \
+	docker buildx use ${DOCKER_BUILDX_BUILDER}; \
+	docker buildx build \
+		--platform $$GOOS/$$GOARCH \
+		-t $$BINARY_NAME \
+		--load \
+		.; \
+	mkdir -p ${RELEASE_FOLDER}; \
+	docker create -ti --name tmp-okp4d $$BINARY_NAME; \
```

Comment:
🧐 if for some reason the `makefile` fails just before removing the created docker image, then the `makefile` will be broken. I guess it would be more robust to test the existence of the image here to take the right steps.